test/distributed/test_device_mesh.py

```diff
@@ -292,6 +292,12 @@ def test_get_item(self):
         dp_group_idx = self.rank % 4
         self.assertEqual(mesh_2d["DP"].mesh, pg_ranks_by_dim_name["DP"][dp_group_idx])
 
+
+class TestMeshEnv(DTensorTestBase):
+    @property
+    def world_size(self):
```

Comment:
why we need this to be 8 world size? I guess it's fine as we will run cpu tests, but we should keep size < 4 unless we really need to bump it so that we can also run tests with 4 gpu CI